aries-site/src/pages/templates/forms/managing-child-objects.mdx

```diff
@@ -0,0 +1,110 @@
+  EditRole,
+} from '../../../examples/templates/forms';
+
+This pattern improves productivity by focusing a user's attention solely on the current
+object of interest, and uses collapsible objects to show or hide detail as needed. The
+intent is that collapsible objects reduce noise, clutter, and cognitive overload
+by hiding extraneous detail &mdash; and an easy means to view detail when needed.
+
+## When to use
+
+Use when a user's primary task is to create or configure a parent resource, but creating,
+updating, or removing child objects is a sub-task. This pattern provides users easy control
+over displaying detail as needed.
+
+## Anatomy
+
+The following illustrates the elements composing a child object in its collapsed and expanded
+states. View the [examples](#examples) to see how child objects behave and are presented in a form
+context.
+
+### Collapsed state
+
+<Example plain>
+  <CollapsedStateAnatomy />
+</Example>
+
+|                 Label                  | Region             | Purpose                                                    |                  Required                   | Notes                                                                                                                                                                |
+| :------------------------------------: | ------------------ | ---------------------------------------------------------- | :-----------------------------------------: | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+|                 **1**                  | **Container**      | Identfies the child and controls the show detail behavior. | <Checkmark a11yTitle="true" size="small" /> | The entirety of the container is focusable and expands to show detail with a mouse click, Enter key, or Spacebar. The aria-label should read "Show and edit detail." |
+|                 **2**                  | **Label**          | Uniquely identifies each child object.                     | <Checkmark a11yTitle="true" size="small" /> | Should be implemented as a semantically correct heading to allow screen reader users to traverse between children.                                                   |
+|                 **3**                  | **Expand icon**    | Visual treatment indicating show detail behavior.          | <Checkmark a11yTitle="true" size="small" /> | Icon name is "FormDown".                                                                                                                                             |
+|                 **4**                  | **Values summary** | A preview summarizing the object's detail.                 |                  Optional                   |                                                                                                                                                                      |
```

Comment:
for values summary should we include when to truncate? should they use grommet `truncation` or is there a recommended length?

Reply:
Grommet truncation. I can clarify.

Reply:
okay maybe we can add a small comment saying to should be implemented as grommets `text` with the `truncation` prop to allow for correct behavior. (something along those lines)